remote-method/index.js

```diff
@@ -91,14 +91,16 @@ module.exports = yeoman.generators.Base.extend({
         message: 'Enter the remote method name:',
         required: true,
         default: name,
-        validate: checkPropertyName
+        validate: checkRemoteMethodName
       },
       {
         name: 'isStatic',
         message: 'Is Static?',
-        required:true,
+        required: false,
         type: 'confirm',
-        default: true
+        default: function(answers) {
+          return !answers.methodName.match(/^prototype\.(.*)$/);
```

Comment:
If we allow `methodName` to start with `prototype.` prefix, then I think we need to remove that prefix before we pass the name to `ModelMethod` constructor. See https://github.com/strongloop/loopback-workspace/pull/273#discussion_r61563760